Mars_Application_UserStory2_TestAutomation/Pages/ProfilePage.cs

```diff
@@ -0,0 +1,213 @@
+﻿using Microsoft.VisualBasic.FileIO;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Assert = Xunit.Assert;
+
+namespace Mars_Application_UserStory2_TestAutomation.Pages
+{
+    public class ProfilePage
+    {
+        public void CreateLanguage(IWebDriver driver, string language, string languageLevel)
```

Comment:
please do not pass the driver around the action methods 

CreateLanguage(IWebDriver driver, string language, string languageLevel)

should be 

CreateLanguage(string language, string languageLevel)

you will either need to extend to Common driver class or you will need to pass driver via the constructor 

private IWebDriver driver,
ProfilePage(IWebDriver driver){

this.driver =driver;
}